backend/api/Controllers/MissionSchedulingController.cs

```diff
@@ -74,19 +78,25 @@ ICustomMissionSchedulingService customMissionSchedulingService
         )
         {
             var robot = await _robotService.ReadById(scheduledMissionQuery.RobotId);
-            if (robot is null)
-            {
-                return NotFound($"Could not find robot with id {scheduledMissionQuery.RobotId}");
-            }
+            if (robot is null) { return NotFound($"Could not find robot with id {scheduledMissionQuery.RobotId}"); }
 
             var missionDefinition = await _missionDefinitionService.ReadById(scheduledMissionQuery.MissionDefinitionId);
-            if (missionDefinition == null)
-            {
-                return NotFound("Mission definition not found");
-            }
+            if (missionDefinition == null) { return NotFound("Mission definition not found"); }
+
+            try { await _localizationService.EnsureRobotIsOnSameInstallationAsMission(robot, missionDefinition); }
```

Comment:
Scheduling Echo missions in the `Create` function below in this file should also ensure that the robot is localized, right?

Reply:
No, currently we have no functionality to say which area an Echo mission is in. For now the Echo mission scheduling will be removed (in the frontend) until we fix that and then we will also include localization for those missions.